### ISSUE TITLE
404 error: favicon.ico not found

### ISSUE THREAD

**Issue by DivideBySara** [britinimalloy/portfolio#18]
This should be fixed for compatibility with accessibility equipment. It could be fixed in 1 of 2 ways:
A) Make your portfolio a Node.js project.
B) Add a file called favicon.ico in your root directory & link to it in the <head> of your index.html page

**Comment by britinimalloy** [comment]
changed to a node.js project and this is now fixed! no errors in console!
